src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/DeferredBuildTest.cs

```diff
@@ -0,0 +1,95 @@
+﻿using NUnit.Framework;
+using System.IO;
+using Xamarin.ProjectTools;
+
+namespace Xamarin.Android.Build.Tests
+{
+	[Parallelizable(ParallelScope.Children)]
+	public class DeferredBuildTest : BaseTest
+	{
+		[Test]
+		public void SelectivelyRunUpdateAndroidResources()
+		{
+			var path = Path.Combine("temp", TestName);
+			var app = new XamarinAndroidApplicationProject
+			{
+				ProjectName = "MyApp",
+			};
+
+			app.SetProperty("AndroidUseManagedDesignTimeResourceGenerator", "True");
+			app.SetProperty("AndroidUseIntermediateDesignerFile", "True");
+
+			using (var appBuilder = CreateApkBuilder(Path.Combine(path, app.ProjectName)))
+			{
+				appBuilder.Target = "Compile";
+				Assert.IsTrue(appBuilder.Build(app, parameters: new string[]
+				{
+                    // Normal DTB properties
+                    "DesignTimeBuild=true",
```

Comment:
looks like the code formatting missed a few spots.

Reply:
I think I fixed the remaining ones... sigh...